<launch>
	<master auto="start"/>

	<!-- Run the map server -->
	<node name="map_server" pkg="map_server" type="map_server" args="$(find launch)/kit_navi/kit_map/yatukaho.yaml my_map_resolution"/>	
	

	<!--Run AMCL -->
	<include file="$(find launch)/kit_navi/amcl_diff.launch" />


	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
		<rosparam file="$(find launch)/kit_navi/kit_yaml/costmap_common_params.yaml" command="load" ns="global_costmap" />
		<rosparam file="$(find launch)/kit_navi/kit_yaml/costmap_common_params.yaml" command="load" ns="local_costmap" />
		<rosparam file="$(find launch)/kit_navi/kit_yaml/local_costmap_params.yaml" command="load" />
		<rosparam file="$(find launch)/kit_navi/kit_yaml/global_costmap_params.yaml" command="load" />
		<rosparam file="$(find launch)/kit_navi/kit_yaml/base_local_planner_params.yaml" command="load" />
	</node>

</launch>
